app/Services/MoviesService.php

```diff
@@ -38,6 +40,8 @@ public function findById($id, $user): Movie
                 return $query->where('user_id', $user->id);
```

Comment:
mozda ceo ovaj query za izvlacenje u posebnu funkciju pa pozvati na oba mesta? Posto vecina stvari je isto, sve ovo sa withCount, with, bez where i firstOrFail, pa vracas query builder iz funkcije i ove obe koriste to